src/System.Management.Automation/resources/SessionStateStrings.resx

```diff
@@ -296,6 +296,9 @@
   </data>
   <data name="ClearContentProviderException" xml:space="preserve">
     <value>Attempting to perform the ClearContent operation on the '{0}' provider failed for path '{1}'. {2}</value>
+  </data>
+    <data name="ClearDirectoryContent" xml:space="preserve">
```

Comment:
It would be good to fix line indent for opening `data` node.

Reply:
Sure, I'm fan of Perfection :-)